docs/contributing/development/continuous_integration.rst

```diff
@@ -10,270 +10,22 @@ or a change is merged into the *master* branch, a service will
-
-Also, variables are available to scripts through environment variables.
-The name is upper-cased and ``.``  is replaced with ``_``. For example
-
-.. code-block:: yaml
-
-  variables:
-    my.var: 'foo'
-
-  steps:
-    - bash: |
-        echo $MY_VAR
-
-To set a variable from a script task, use the ``task.setvariable`` logging
-command.
-
-.. code-block:: yaml
-
-  steps:
-
-    - bash: |
-        echo "##vso[task.setvariable variable=my.var]foo"
-
-    - bash: |
-        echo $(my.var)
-
-See the `Azure documentation section on variables`_ for more information.
-
-
-Predefined variables
---------------------
-
-The most important (and confusing) predefined variables are the ones related
-to paths in Azure:
-
-* All folders for a given pipeline are created under ``Agent.BuildDirectory`` 
-  variable, alias ``Pipeline.Workspace``. This includes subdirectories like
-  ``/s`` for sources or ``/a`` for artifacts.
-
-* Path to source code varies depending on how many repositories we fetch.
-  For example, source code is located under the ``Build.Repository.LocalPath``
-  variable (alias ``Build.SourcesDirectory``) when fetching a single repository,
-  but after fetching a second repository code is moved automatically to
-  ``Build.Repository.LocalPath/repository-name``.
-
-See the Azure documentation to learn more about `checking out multiple repositories`_.
-
-
-Jobs
-----
-
-You can organize your pipeline into jobs. Every pipeline has at least one job.
-A job is a series of steps that run sequentially as a unit. In other words,
-a job is the smallest unit of work that can be scheduled to run.
-
-
-.. code-block:: yaml
-
-  jobs:
-  - job: myJob
-
-    pool:
-      vmImage: 'ubuntu-latest'
-
-    steps:
-    - bash: echo "Hello world"
-
-Jobs can run in parallel (for example: run the same job on multiple OSes) or
-depend on a previous job.
-
-See the `Azure documentation section on jobs`_ for more information.
-
+Currently, we use GitHub Actions to run all of our pipelines. Making changes to an existing 
+pipeline is as easy as making a pull request. To create a new GitHub Action workflow, 
+just create a new YAML file in ``.github/workflows``.
 
 TARDIS Pipelines
 ----------------
 
-Brief description of pipelines already implemented on Azure or GitHub Actions.
-
-
-The default template
-====================
-
-Templates let you define reusable content, logic, and parameters. It functions
-like an include directive in many programming languages (content from one file
-is inserted into another file).
-
-The common set of steps used across most TARDIS pipelines resides in the
-"default" template:
-
-- Force ``set -e`` on all Bash steps.
-- Set TARDIS custom variables.
-- Fetch TARDIS main repository.
-- Fetch TARDIS reference data repository from mirror (optional).
-- Configure Anaconda for Linux and macOS agents.
-- Install Mamba package manager (optional).
-- Install TARDIS environment (optional).
-- Build and install TARDIS (optional).
-
```

Comment:
Can you put in for example, a common set of steps would be use the setup_lfs action if you need access to the regression/atomic data, use setup_env action to setup the environment and use this?- 
defaults:
  run:
    shell: bash -l {0}